src/inc.js

```diff
@@ -14,4 +15,4 @@ var add = require('./add');
  *
  *      R.inc(42); //=> 43
  */
-module.exports = add(1);
+module.exports = _curry1(add(1));
```

Comment:
This is necessary because `R.add(1)()` does not throw, but `R.inc()` must throw to satisfy the first invariant.

Were we to replace the first invariant with one that states that for any function `f` where `f.length > 0`, `f()` is equivalent to `f`, we wouldn't need the `_curry1` decorator here.